flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/SchedulerTestingUtils.java

```diff
@@ -213,4 +267,161 @@ public static CheckpointCoordinator getCheckpointCoordinator(SchedulerBase sched
 			return operatorGateway.sendOperatorEventToTask(task, operator, evt);
 		}
 	}
+
```

Comment:
There are more than two empty lines. Since this is forbidden by checkstyle in production code, I also wouldn't do it in test code.

Reply:
Agreed. Just did it by mistake.

Reply:
fixed